source/modules/assets/phones.rst

```diff
@@ -1,83 +1,239 @@
 
-.. include:: tabs/components.rst
 
-.. include:: tabs/connexions.rst
+Connections
+-----------
 
-.. include:: tabs/network-ports.rst
+It is possible to create a :doc:`connections <../../tabs/connections>` with a telephone using the following element:
 
-.. include:: ../tabs/management.rst
+* `Computer <computers.html>`_
 
-.. include:: ../tabs/contracts.rst
 
-.. include:: ../tabs/documents.rst
+Network Ports
+-------------
 
-.. include:: ../tabs/tickets.rst
+This tab allows to manage the :doc:`network ports <../../tabs/network_ports>` attached to an equipment.
+The information that can be viewed is:
 
-.. include:: ../tabs/problems.rst
+* Name
+* Port number
+* MTU
+* Speed
+* Internal status
+* Last change
+* Number of I/O bytes
+* Number of I/O errors
+* Duplex
+* VLAN
+* Connected to
+* Connection
+* Deleted
 
-.. include:: ../tabs/external-links.rst
 
-.. include:: ../tabs/notes.rst
+Sockets
+-------
 
-.. todo::
-   Fix this reference (include or link???)
-   **[Tab "Reservations"](Les_différents_onglets/Onglet_Réservations.rst)**
-   Management of reservations for an inventory object
+:doc:`Sockets <../../tabs/sockets>` are the list of physical sockets present on the hardware. These sockets can be Ethernet, USB, HDMI, etc.
+This information cannot be returned by the automatic inventory, so you have to add it manually.
 
-.. include:: ../tabs/historical.rst
+It enables hardware to be linked by cables. Socket is also linked to the :doc:`cables <../modules/assets/cables>` object
 
-.. include:: ../tabs/debug.rst
+Remote management
+-----------------
 
-.. include:: ../tabs/all.rst
+:doc:`Remote management <../../tabs/remote-management>` is used to reference the remote access software installed on the workstation, such as Teamviewer, Anydesk, etc.
+It is possible to add software manually if required, but the information can be fed back via the automatic inventory.
+
+Management
+----------
+
+:doc:`Management <../../modules/tabs/management>` of financial and administrative information, this information is visible in the 'Management' tab on the computer's form.
+
+Contracts
+---------
+
+GLPI supports :doc:`contracts <../management/contract>` management, in order to manage contract types such as loan, maintenance, support...
+
+Contracts management allows to:
+
+* make an inventory of all contracts related to the organization assets
+* integrate contracts in GLPI financial management
+* anticipate and follow contract renewal.
+
+
+Documents
+---------
+
+The :doc:`document <../management/documents>` tab lets you link different types of file to a material (PDF, txt, png, etc.)
+You can attach a document already uploaded to GLPI or add a new one directly from this tab.
+
+Antiviruses
+-----------
+
+:doc:`Antiviruses <tabs/antivirus>` lists all the antivirus programs detected on the computer.
+For Windows, it must be detected in the Windows Security Center in order to be listed in this tab.
```

Comment:
This description doesn't make sense for phones. No need to mention Windows since nobody makes a Windows-based phone anymore.

```suggestion
:doc:`Antiviruses <tabs/antivirus>` lists all the antivirus programs detected on the phone.
```